tests/publiccloud/az_l8s_nvme.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test nvme performance on a L8s_v2 Azure instance
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+# Benchmark the given device with hdparm and fail, if the given threshold [MB/sec] is not reached
+sub benchmark_device {
+    my ($device, $threshold) = @_;
+
+    my $output = script_output("hdparm -t $device | grep 'Timing buffered disk reads'");
```

Comment:
how long does this command take? Do we need timeout?

Reply:
No this takes only a couple of seconds, so in all sane configuration it should be well below the typical timeout of 30 seconds.

Reply:
Ok, I thought this was doing some performance stuff :) Good